src/microbe_stage/MicrobeStage.cs

```diff
@@ -827,6 +827,7 @@ protected override void SpawnPlayer()
             OnIngestedByHostile = OnPlayerEngulfedByHostile,
             OnSuccessfulEngulfment = OnPlayerIngesting,
             OnEngulfmentStorageFull = OnPlayerEngulfmentLimitReached,
+            OnEngulfmentStorageNotFullAnymore = OnPlayerEngulfmentNotFullAnymore,
```

Comment:
This is not "not full anymore", I'd say the name should be `OnEngulfmentStorageNearlyEmpty` to match how this is triggered now.